tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -40,20 +42,22 @@ bool IsRetriable(Status status) {
 Status CallWithRetries(const std::function<Status()>& f) {
```

Comment:
Did you want to add 'initial_delay_seconds' as a parameter to pass it to CallWithRetries? Otherwise it's not clear to me how initial_delay_seconds inside CallWithRetries is available, or am I missing something?